src/main/java/heylo/fileHandling/FilesReader.java

```diff
@@ -0,0 +1,46 @@
+package heylo.fileHandling;
+
+import heylo.tasks.Task;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Handles reading from files.
+ */
+public class FilesReader {
```

Comment:
Suggestion: FileReader might be a better name than FilesReader since this class only reads one file at a given time. Similar comment for FileWriter as well.